Parsers/ASimAuditEvent/Tests/SentinelOne_AsimAuditEvent_SchemaTest.csv

```diff
@@ -0,0 +1,74 @@
+﻿Result
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [ObjectId]"
```

Comment:
can we map this?
The id of the object on which the operation identified by EventType is performed.